client-app/src/desktop/tabs/examples/ExamplesTab.js

```diff
@@ -1,138 +1,230 @@
-                    a {link('Remote API', 'https://newsapi.org/')}. Refresh rate, news sources, and API key can
-                    be modified in the Admin Config tab.
-                </p>,
-                <p>
-                    On the client side, we use a {link(code('DataView'), '../app/grids/dataview')} grid
-                    to support custom filtering logic and rich component rendering.
-                </p>
-            ]
-        },
-        {
-            title: 'FDA Recalls',
-            icon: Icon.health(),
-            path: '/recalls',
-            text: [
-                <p>
-                    This applet uses the openFDA drug enforcement reports API, which provides information on drug recall
-                    events since 2004. Provides examples of filtering and searching data from an external API.
-                </p>,
-                <p>
-                    For more information, see {link('here', 'https://open.fda.gov/apis/drug/enforcement/')}.
-                </p>
-            ]
-        },
+
+const sideBar = hoistCmp.factory(({model}) => {
+    return toolbar({
+        className: 'app-toolbar',
+        items: [...model.examples.map(e => button({
+            icon: e.icon,
+            onClick: () => model.activeApp === e.title ? window.open(e.path) : model.setActiveApp(e.title)
+        })),
+        button({
+            icon: Icon.chevronRight(),
+            onClick: () => model.leftPanelModel.collapsed = false
+        })],
+        vertical: true
+    });
+});
+
+class LocalModel extends HoistModel {
+
+    @bindable activeApp = 'Portfolio';
+
+    @managed
+    leftPanelModel = new PanelModel(
         {
-            title: 'File Manager',
-            icon: Icon.fileArchive(),
-            path: '/fileManager',
-            text: [
-                <p>
-                    This example shows a simple, full-stack pattern for syncing files to a server.
-                </p>,
-                <p>
-                    On the client side this app uses the {link(code('FileChooser'), '/app/other/fileChooser')}.
-                    The server-side controller and service provide examples of how uploads can
-                    be extracted from the request and processed within Grails.
-                </p>,
-                <p>
-                    <strong>This example is visible only to admins</strong> to avoid
-                    arbitrary file uploads to our server.
-                    Please {link('contact us', 'https://xh.io/contact/')} for access.
-                </p>
-            ]
+            defaultSize: 300,
+            collapsible: true,
+            resizable: false,
+            side: 'left'
         }
-    ];
+    )
+
+    get examples() {
+        return [
+            {
+                title: 'Portfolio',
+                icon: Icon.portfolio(),
+                path: '/portfolio',
+                cmp: () => portfolioPanel(),
```

Comment:
this cmp argument is cruft?  We aren't doing this anymore -- the main point of using our iFrame